code/game/objects/items/stacks/sheets/sheets.dm

```diff
@@ -45,3 +45,134 @@
 	user.visible_message(span_notice("[user] shatters the sheet of [name] on the floor, leaving [english_list(shards)]."), \
 		span_notice("You shatter the sheet of [name] on the floor, leaving [english_list(shards)]."))
 	return TRUE
+
+/obj/item/stack/sheet/mineral/reagent
+	name = "reagent ingots"
+	desc = "Ingots made out of treated solidified reagents"
+	singular_name = "reagent ingot"
+	icon_state = "sheet-silver"
+	mats_per_unit = list(/datum/material/custom=MINERAL_MATERIAL_AMOUNT)
+	merge_type = /obj/item/stack/sheet/mineral/reagent
+	amount = 1
+	max_amount = 50
+	material_type = /datum/material/custom
+	var/datum/reagent/reagent_type
+
+
+GLOBAL_LIST_INIT(reagent_recipes, list (
+	new /datum/stack_recipe("reagent door", /obj/structure/mineral_door/transparent/reagent, 5, one_per_turf = 1, on_floor = 1),
```

Comment:
there a reason why reagent walls and shit are being added in the same PR, what is the significance of even allowing this?

Reply:
The significance is ingots -> walls/doors/floors.